README.md

```diff
@@ -74,11 +74,18 @@ Valhalla uses the [GNU Build System](http://www.gnu.org/software/automake/manual
     #if you plan to compile with python bindings, see below for more info
     sudo apt-get install -y python-all-dev
 
+To install on macOS, you need to install its dependencies with [Homebrew](http://brew.sh):
+
+    brew install autoconf automake libtool protobuf-c boost-python libspatialite pkg-config lua czmq
+
 And then run to install it:
 
+    git submodule update --init --recursive
     ./autogen.sh
+    # on macOS you need to tell linkers how to reach home-brewed sqlite3
+    # export LDFLAGS="-L/usr/local/opt/sqlite/lib/ -lsqlite3"
     ./configure
-    make test -j$(nproc)
+    make test -j8
```

Comment:
so yeah mac hardware is mostly the same but you cant really do this on any random pc. ones with single or only dual core will basically be fork bombed not to mention the ram requirement is significantly higher when you do this. if mac doesnt have `nproc` hardcoding isnt an option unless we make it `-j1` which i think we all agree is a bad option. maybe we can just do: 

`grep -c processor /proc/cpuinfo`